roottest/root/io/evolution/CMakeLists.txt

```diff
@@ -1,8 +1,205 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST(LABELS longtest io)
```

Comment:
The test seems to have lost the io label ...

Reply:
> The test seems to have lost the io label ...

I will add labels to all tests.

Reply:
Thanks (albeit no all of the new test should be marked as long :) ).